src/view/moveable/moveable.jsx

```diff
@@ -89,9 +89,8 @@ export default class Movable extends Component<Props> {
       // Expecting a flow error
       // React Motion type: children: (interpolatedStyle: PlainStyle) => ReactElement
       // Our type: children: (Position) => (Style) => React.Node
-      // $ExpectError
       <Motion defaultStyle={origin} style={final} onRest={this.onRest}>
-        {(current: Position) =>
+        {(current: { [string]: number }) =>
```

Comment:
Why can we not use the `Position` type here?

Reply:
Because the real type of this is more common.

Reply:
Because we pull out .x and .y i think that using `Position` here is the most clear about what we are dealing with - as the object will always have only these properties

Reply:
We inherit this type from react-motion, ignoring this won't help. We can take care of react-motion types later.